tests/e2e/scale/test_pvc_creation_deletion_scale.py

```diff
@@ -1,5 +1,6 @@
 """
-Test to measure pvc scale creation & deletion time. Total pvc count would be 1500
+Test to measure pvc scale creation & deletion time. Total pvc count would be
+500 times the number of worker nodes
```

Comment:
@ramkiperiy with 4.6, it is 1000 PVCs. we should scale up to 1000 PVCs per OCS worker node, isn't it?

Reply:
The 4.6 version was 1500 PVCs tottal.   This works out to 500 PVCs per node.  In later conversations, this was what we decided to set this as.